src/cli/providers/messenger/persona.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable consistent-return */
+import chalk from 'chalk';
+import invariant from 'invariant';
+import { MessengerClient } from 'messaging-api-messenger';
+
+import getConfig from '../../shared/getConfig';
+import { bold, error, print } from '../../shared/log';
+
+const help = () => {
+  console.log(`
+  bottender messenger persona <command> [option]
+
+  ${chalk.dim('Commands:')}
+
+    list              List all personas.
+    create            Create new personas.
```

Comment:
Create a new persona

Reply:
The create command should create only one persona like
`persona create <NAME> <PROFILE_IMAGE_URL>`
or, given an array of personas in config, create by updated different ones like set messenger profile did?